test/BodyParams/BodyParamsMiddlewareTest.php

```diff
@@ -78,7 +78,7 @@ public function notApplicableProvider()
             ['GET', 'application/json'],
             ['HEAD', 'application/json'],
             ['OPTIONS', 'application/json'],
-            ['POST', 'application/x-www-form-urlencoded'],
+            ['GET', 'application/x-www-form-urlencoded'],
```

Comment:
Because this makes POST of `application/x-www-form-urlencoded` an applicable type, thus the test would break if I didn't change it.